cjwstate/modules/message_catalogs.py

```diff
@@ -0,0 +1,116 @@
+from cjworkbench.i18n import supported_locales, default_locale
```

Comment:
Could we please call the module `i18n`?

Underscores in module names are un-Pythonic. I know we have a few. (The most notable is `module_loader`, which really should be `loader`). But we don't need more.